app/views/sites/api_instructions/_show_web_md.html.haml

```diff
@@ -1,485 +1,77 @@
 :markdown
-  <!-- Reminder to update https://open.gsa.gov/api/searchgov-results/ with any changes here. -->
-
-  This API exposes all relevant results “modules” in a single JSON call, including:
-
-
+  ## Overview
 
+  The Search.gov Results API exposes all relevant results “modules” in a single JSON call, including the following:
 
   * Web results;
   * Best bets;
   * Health topics;
   * Job openings;
   * Recent tweets;
-  * Recent news;
-  * Recent video news;
   * Federal Register documents; and
   * Related searches.
 
 
-  ## Note about Web Results and Endpoints
-
+  ## Getting started
 
+  Please refer to the detailed documentation for the Results API, now publicly available through the [OpenGSA Portal](https://open.gsa.gov/api/searchgov-results/). The information below serves as an introduction.
 
-
-  The endpoint you use to retrieve web results through this API will depend on the method we used to index your content. If we don't yet have your content indexed, you won't see results in the API.
-  
   We can index content using your [XML sitemap](https://search.gov/blog/sitemaps.html) (preferred), [RSS feeds](#{site_rss_feeds_path(@site)}), or the [indexing API, i14y](https://search.gov/developer/i14y.html). We can also deploy a crawler on a limited basis.
```

Comment:
Much better than the original. More concise while still retaining all of the information.